corehq/form_processor/tests/utils.py

```diff
@@ -170,6 +172,10 @@ def _delete_all(db, viewname, **view_kwargs):
 )
 
 
+def sql_backend_case(cls):
```

Comment:
why not use `test_sql_backend` or something similar? This name makes me think it's going to return a CommCareCase for a sql domain